bundles/com.espressif.idf.ui/META-INF/MANIFEST.MF

```diff
@@ -30,12 +30,22 @@ Require-Bundle: org.eclipse.core.runtime,
  org.eclipse.ui.console,
  org.eclipse.jgit,
  org.eclipse.swtchart,
- org.eclipse.cdt.native.serial
+ com.google.gson,
+ org.eclipse.nebula.widgets.xviewer;bundle-version="1.1.0",
+ org.eclipse.nebula.widgets.xviewer.core;bundle-version="1.0.0",
+ org.eclipse.nebula.widgets.opal.dialog;bundle-version="1.0.0",
+ org.eclipse.nebula.widgets.opal.dialog.snippets;bundle-version="1.0.0",
+ org.eclipse.nebula.widgets.opal.logindialog;bundle-version="1.0.0",
+ org.eclipse.nebula.widgets.opal.logindialog.snippets;bundle-version="1.0.0",
```

Comment:
Could you check if we need `snippets` plugins and remove them if not needed

Reply:
Done